templates/jfrog-rt6-eks-core-workload.template.yaml

```diff
@@ -233,53 +233,6 @@ Resources:
               partition: artifactory-primary
             service:
               loadBalancerSourceRanges:  [ "${AccessCIDR}" ]
-            artifactoryConf: |
```

Comment:
@vinayagg Isn't this still needed?

Reply:
This was overriding the config from core product. however the values have been merged back into core product and we do not need to override. I have tested and these templates work fine without the override now (in fact they break with the override for some other reasons)